it/pages/TwoEyeReviewerControllerISpec.scala

```diff
@@ -0,0 +1,168 @@
+class TwoEyeReviewerControllerISpec extends IntegrationSpec {
+
+  private val successPayload: JsValue = Json.parse(
+    """{
+      |  "id":"oct90005",
+      |  "title":"Telling HMRC about extra income",
+      |  "lastUpdated":"2020-05-10",
+      |  "pages":[
+      |    {
+      |      "id":"id1",
+      |      "title":"how-did-you-earn-extra-income",
+      |      "status":"NotStarted"
+      |    },
+      |    {
+      |      "id":"id2",
+      |      "title":"sold-goods-or-services/did-you-only-sell-personal-possessions",
+      |      "status":"NotStarted"
+      |    },
+      |    {
+      |      "id":"id3",
+      |      "title":"sold-goods-or-services/have-you-made-a-profit-of-6000-or-more",
+      |      "status":"NotStarted"
+      |    },
+      |    {
+      |      "id":"id4",
+      |      "title":"sold-goods-or-services/have-you-made-1000-or-more",
+      |      "status":"NotStarted"
+      |    },
+      |    {
+      |      "id":"id5",
+      |      "title":"sold-goods-or-services/you-do-not-need-to-tell-hmrc",
+      |      "status":"NotStarted"}
+      |    ,
+      |    {
+      |      "id":"id6",
+      |      "title":"rent-a-property/do-you-receive-any-income",
+      |      "status":"NotStarted"
+      |    },
+      |    {
+      |      "id":"id7",
+      |      "title":"rent-a-property/have-you-rented-out-a-room",
+      |      "status":"NotStarted"
+      |    }
+      |  ]
+      |}""".stripMargin
+  )
+
+  "Calling the approval 2i review endpoint" should {
+
+    "return an Ok response" in {
+
+      AuditStub.audit()
+      AuthStub.authorise()
+
+      ExternalGuidanceStub.approval2iReview(Status.OK, successPayload)
+
+      val request: WSRequest = buildRequest("/2i-review/oct90005")
+
+      val response: WSResponse = await(request.get())
+
+      response.status shouldBe Status.OK
+    }
+
+    "return a not found response when the external guidance service cannot locate the requested process" in {
+
+      AuditStub.audit()
+      AuthStub.authorise()
+
+      val responsePayload: JsValue = Json.toJson(NotFoundError)
+
+      ExternalGuidanceStub.approval2iReview(Status.NOT_FOUND, responsePayload)
+
+      val request = buildRequest("/2i-review/oct90005")
+
+      val response: WSResponse = await(request.get())
+
+      response.status shouldBe Status.NOT_FOUND
+    }
+
+    "return an internal server error response when an unexpected error is returned by the external guidance service" in {
+
+      AuditStub.audit()
+      AuthStub.authorise()
+
+      val responsePayload: JsValue = Json.toJson(InternalServerError)
+
+      ExternalGuidanceStub.approval2iReview(Status.INTERNAL_SERVER_ERROR, responsePayload)
+
+      val request = buildRequest("/2i-review/oct90005")
+
+      val response: WSResponse = await(request.get())
+
+      response.status shouldBe Status.INTERNAL_SERVER_ERROR
+    }
+
+    "return an internal server error response when the external guidance service returns a stale data error" in {
```

Comment:
I know we said the code won't process stale data errors, but shouldn't we show Not Found page instead of internal error

Reply:
I have updated the test to check for a not found error.